setup.cfg

```diff
@@ -21,17 +19,15 @@ include_package_data = True
 packages = find:
 zip_safe = False
 test_suite = tests
-python_requires = >= 2.7, != 3.0.*, != 3.1.*, != 3.2.*, != 3.3.*, != 3.4.*
+python_requires = >= 3.6
 setup_requires =
     setuptools
 install_requires =
     jsonschema
-    openapi-schema-validator
+    openapi-schema-validator @ git+https://github.com/anexia-it/openapi-schema-validator@master
```

Comment:
This should be reverted once https://github.com/p1c2u/openapi-schema-validator/pull/18 is completed and a new release is available

Reply:
updated to `openapi-schema-validator>=0.3.0a1`